LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020-2021 Poimandres
+Copyright (c) 2020-2022 Poimandres
```

Comment:
```suggestion
Copyright (c) 2020 Poimandres
```
I don't think we need the range, so this is fine.
Anyway, merging this as it's harmless. It may look active.